packages/wallet/src/account.test.ts

```diff
@@ -103,6 +103,22 @@ describe('Account', () => {
     // #endregion Message-getResourcesToSpend
   });
 
+  it('getResourcesToSpend should work with <1 amount', async () => {
+    // #region Message-getResourcesToSpend
```

Comment:
Seems we now have 2 regions called `Message-getResourcesToSpend`, potentially copied from the above test?

Reply:
Good catch! Indeed, it was a copy-paste.